contrib/polarity-for-haskellers/README.md

```diff
@@ -0,0 +1,115 @@
+# Polarity for Haskellers
+
+It might be a bit confusing to understand what polarity's `data` and `codata` and `def` and `codef` do differently than Haskell's data. Especially observing infinite streams is incredibly easy in Haskell.
```

Comment:
```suggestion
It might be a bit confusing to understand what Polarity's `data` and `codata` and `def` and `codef` do differently than Haskell's data. Especially observing infinite streams is incredibly easy in Haskell.
```